data/mods/Magiclysm/npc/classes.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "type": "npc_class",
+    "id": "NC_MAGE",
```

Comment:
can you make this more specific? he looks like a novice, or maybe average stormshaper.

Reply:
tbh that was just for the purposes of testing, that particular NPC would not show up anywhere, and Id expect that contributors who made actual NPCs for magiclysm would make more specific and sane classes, what I can do is rename it to NC_MAGE_TEST or something